tests/cases/contrib/@xml:base.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../jats.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../../src/support.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<contrib contrib-type="author" xml:base="http://example.com/"/>
```

Comment:
Should we require a `<name>`? Would match https://github.com/libero/browser/pull/87#discussion_r294239841 which ignores it if it doesn't find one.